packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx

```diff
@@ -130,21 +136,24 @@ function CustomViewLoader(props: TCustomViewLoaderProps) {
   ].join('/');
 
   return (
-    <Drawer
```

Comment:
Ah so we can't use the Drawer component anymore?

Reply:
Not in this case.
The `Drawer` component will always use a header which is something we don't want in this use case.